### PR TITLE
Fix cart drawer for variant list and tablet spacing

### PR DESCRIPTION
**Pull request by sofiamatulis**
### PR Summary: 

- Cart drawer wasnt re-rendering when the items were being added to cart from the variant list. Video: https://screenshot.click/20-08-umlhd-pod07.mp4

I am re-rendering the section to ensure this happens

- Fix spacing for tablet: https://screenshot.click/20-53-zjc44-6aakj.png

Editor: https://admin.shopify.com/store/os2-demo/themes/140319490070/editor

### CODE REVIEW

**Comment by KaichenWang** [comment]
> Fix spacing for tablet: https://screenshot.click/20-53-zjc44-6aakj.png


This change doesn't seem to be included in this PR currently

**Comment by sofiamatulis** [comment]
> > Fix spacing for tablet: https://screenshot.click/20-53-zjc44-6aakj.png
> 
> This change doesn't seem to be included in this PR currently

Sorry my last commit didnt go through! https://github.com/Shopify/dawn/pull/2880/commits/86abc40eede73abb9d5bf51b10f0d031b0bcb1c5

File: assets/quick-order-list.js [hunk 1]
```diff
@@ -150,19 +151,33 @@ class QuickOrderList extends HTMLElement {
...
 
   renderSections(parsedState) {
     this.getSectionsToRender().forEach((section => {
       const sectionElement = document.getElementById(section.id);
+      if (sectionElement.parentElement.classList.contains('drawer')) {
+        parsedState.items.length > 0 ? sectionElement.parentElement.classList.remove('is-empty') : sectionElement.parentElement.classList.add('is-empty');
+
+        setTimeout(() => {
+          document.querySelector('#CartDrawer-Overlay').addEventListener('click', this.cart.close.bind(this.cart));
+          this.cart.open();
```
[the remark this answers is not in the document]
Comment (sofiamatulis) [reply assets/quick-order-list.js:171]: cc @edmund-teh @nicklepine thoughts? that means we can include Meli's suggestion earlier (you can see the behaviour on the PR)

However, that means when you add things and delete things it opens every time (if you have drawer selected).

Popup and page remains the same

Comment (eugenekasimov) [inline assets/quick-order-list.js:171]: I approve, but let's account for what Ed and Nick say about this 👆 looks a bit annoying to see a cart drawer every time.

**Review by eugenekasimov — APPROVED** [review]

File: assets/quick-order-list.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nicklepine) [reply assets/quick-order-list.js:171]: The cart drawer should not open when you make a quantity update.

[the remark this answers is not in the document]
Comment (sofiamatulis) [reply assets/quick-order-list.js]: Updated!

**Pull request merged**